CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.2.10
+ * Look for GCE metadata host in environment.
```

Comment:
Document the environment variable by name, ie. write `$GCE_METADATA_HOST`

Reply:
Done